### PR TITLE
Project Capstone 

### PR DESCRIPTION
**Pull request by mwanawabangona**
- Add html Css
- Add Linters
- Add JavaScript
- Add README file

### CODE REVIEW

File: home.css [hunk 1]
```diff
@@ -0,0 +1,660 @@
...
+.partner-list {
+  width: 100%;
+}
+
+.partner-image {
+  width: 125px;
+  height: 40px;
+}
+
+footer {
```
Comment (MahmoudBakr23) [inline home.css:381]: - [ ] As in the project requirements. Kindly make the background of the footer dark in the desktop version 🙏

File: home.css [hunk 2]
```diff
@@ -0,0 +1,660 @@
...
+  margin-top: 5%;
+}
+
+h3 {
+  font-family: 'cocogoose', sans-serif;
+  font-size: 1.84rem;
+  color: #504f4f;
+}
+
+.venue {
```
Comment (MahmoudBakr23) [inline home.css:193]: - [x] On the mobile version, this element is causing a white gap between the main section and the program section, please fix that.
_See the image down below._
Screenshot

File: home.css [hunk 3]
```diff
@@ -0,0 +1,660 @@
...
+    padding-inline: 0.8%;
+    cursor: pointer;
+  }
+
+  header {
+    background-color: #fff;
+    height: 3rem;
+  }
+
+  .navigation-container {
```
Comment (MahmoudBakr23) [inline home.css:443]: - [x] In the medium-sized screen there's a break in the second bar at both home and about pages, please fix them 😄 
_See images down below._
Screenshot1
Screenshot

File: page.css [hunk 4]
```diff
@@ -0,0 +1,523 @@
...
+
+.flex-past h2 {
+  text-align: center;
+}
+
+.flex-past p {
+  text-align: center;
+}
+
+.flex-past img {
```
Comment (MahmoudBakr23) [inline page.css:245]: - [x] On the medium-sized screen, the two images are not consistent, please let's work on that. They're supposed to be aligned horizontally. 😄 
_See the image down below._
Screenshot3

File: about.html [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+      <a href="#"> My Logo</a>
+    </div>
+    <p class="copy-right">
+      2021 NBA Summit Miami <br> SomeRights Reserved
+    </p>
+  </div>
+  </footer>
+
+
+
```
Comment (MahmoudBakr23) [inline about.html]: - [x] For best practices, please remove any extra blank lines or indentations for cleaner and more optimized code. 🙏 😄 
_You can find more about this tip and other useful ones at this [link](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)._

File: js/menu.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+const closeMenu = document.getElementById('close');
+const menu = document.getElementById('mob-nav');
+
+function openMobileMenu() {
+  menu.style.display = 'block';
+  openMenu.style.display = 'none';
+  closeMenu.style.display = 'block';
+}
+
+function closeMobileMenu() {
```
Comment (MahmoudBakr23) [inline js/menu.js:11]: - [ ] When I open the menu and close it, the hamburger button doesn't disappear even when I switch to the desktop version. Please double-check on that! 😄 
_See the marked image down below._
Screenshot

File: js/data.js [hunk 7]
```diff
@@ -0,0 +1,46 @@
+const data = [
```
Comment (MahmoudBakr23) [inline js/data.js:1]: - [ ] I see that you declared the `featured speakers` section's data right here in this file but you never used it in another place, and you wrote the exact same data object in the `sectrion.js` file, would you please tell me what you got in mind? 😄 🙏 
However, I highly recommend removing this extra file if it's not used for more optimized and organized code.

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @mwanawabangona 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Linter is correct
- Descriptive README file, PR headline, summary, and commit messages
- Mobile-first approach
- Valid live demo link
- Great code separation
- Well done dynamically adding the featured speakers' section

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️

_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] I recommend attaching screenshots of your app in the README file 😄 

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: home.css [hunk 8]
```diff
@@ -0,0 +1,674 @@
...
+  z-index: 100;
+}
+
+.logo-desktop a {
+  position: absolute;
+  top: 1.3rem;
+  right: 1.5rem;
+  text-decoration: none;
+  color: black;
+}
```
Comment (dasileker) [inline home.css:70]: - [ ] the `logo` and the menu in the `nav` is not displaying correctly in the mobile version
log

File: home.css [hunk 9]
```diff
@@ -0,0 +1,674 @@
...
+
+.fa-bars {
+  font-size: 2rem;
+}
+
+.first-section {
+  background-image: url(assets/nba9.jpg);
+  background-repeat: no-repeat;
+  background-position: center;
```
Comment (dasileker) [inline home.css:153]: - [ ] Also the background image is kind of not fitting in the mobile view
bb

**Review by dasileker — CHANGES_REQUESTED** [review]
**### Hi @mwanawabangona** 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: home.css (hunk 8 above)
Comment (rloterh) [reply home.css:70]: This issue is well resolved ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Ellis, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾 🍾 :shipit: .
Congratulations! 🎉 🎉 🌟 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make you

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EricMbouwe — APPROVED** [review]
### Hi @mwanawabangona, 

# Status: Approved ✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**